package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -365,7 +365,7 @@ def _write_velocities(self, atoms):
         )
         for index, vel in zip(indices, velocities):
             self.f.write(
-                "{i:d} {x:f} {y:f} {z:f}\n".format(
+                "{i:d} {x:.8f} {y:.8f} {z:.8f}\n".format(
```

Comment:
This may be useful but does not belong into this PR. Please revert and make a separate PR – then people knowledgable about LAMMPS can discuss.